ui/page/staking/overview.go

```diff
@@ -446,6 +489,124 @@ func (pg *Page) HandleUserInteractions() {
 	if clicked, selectedItem := pg.ticketsLive.ItemClicked(); clicked {
 		pg.ChangeFragment(tpage.NewTransactionDetailsPage(pg.Load, pg.liveTickets[selectedItem].transaction))
 	}
+
+	_, walID, _, _ := pg.WL.MultiWallet.GetAutoTicketsBuyerConfig()
+	if pg.autoPurchase.Changed() {
+		if pg.autoPurchase.IsChecked() {
+			if pg.WL.MultiWallet.TicketBuyerConfigIsSet() {
+				pg.startTicketBuyerPasswordModal()
+			} else {
+				newTicketBuyerModal(pg.Load).
+					CancelSave(func() {
+						pg.autoPurchase.SetChecked(false)
+					}).
+					SettingsSaved(func() {
+						pg.startTicketBuyerPasswordModal()
+						pg.Toast.Notify("Auto ticket purchase setting saved successfully")
+					}).
+					Show()
+			}
+		} else {
+			pg.autoPurchase.SetChecked(false)
+			go pg.WL.MultiWallet.StopAutoTicketsPurchase(walID)
+		}
+	}
+
+	if pg.autoPurchaseSettings.Button.Clicked() {
+		if pg.WL.MultiWallet.IsAutoTicketsPurchaseActive(walID) {
+			return
+		}
```

Comment:
Should this button be disabled if TB is currently active? Or show a toast message that settings cannot be changed while TB is active, so the user disables TB before trying to change the settings.

Reply:
It does make me wonder, can only one wallet be used for auto ticket purchases at a time? Should probably add a TODO to extend this to multiple wallets subsequently if that's the case.

Reply:
Looking at the screenshots, I don't think we need this button since it does nothing if auto tb is already active. Instead, if it is not active and the user wants to enable it, show the settings modal with previously set values so the user can just proceed with the settings unchanged or change the settings to what they want now and proceed to enter password. The user may not always remember their last configured setting, and it would be more work if a user wanted to edit their settings before running auto tb - they'll have to first disable auto tb, click settings, update, save, then click the checkbox to resume tb.